locales/sv-SE.json

```diff
@@ -0,0 +1,336 @@
+    "View YouTube comments": "Visa YouTube-kommentarer",
+    "View more comments on Reddit": "Visa flera kommentarer på Reddit",
+    "View `x` comments": "Visa `x` kommentarer",
+    "View Reddit comments": "Visa Reddit-kommentarer",
+    "Hide replies": "Dölj svar",
+    "Show replies": "Visa svar",
+    "Incorrect password": "Fel lösenord",
+    "Quota exceeded, try again in a few hours": "Kvoten överskriden, försök igen om ett par timmar",
+    "Unable to log in, make sure two-factor authentication (Authenticator or SMS) is turned on.": "Kunde inte logga in, försäkra dig om att tvåfaktors-autentisering (Authenticator eller SMS) är påslagen.",
+    "Invalid TFA code": "Ogiltig tvåfaktor-kod",
+    "Login failed. This may be because two-factor authentication is not turned on for your account.": "Inloggning misslyckades. Detta kan vara för att tvåfaktors-autentisering inte är påslaget på ditt konto.",
+    "Wrong answer": "Fel svar",
+    "Erroneous CAPTCHA": "Ogiltig CAPTCHA",
+    "CAPTCHA is a required field": "CAPTCHA är ett obligatoriskt fält",
+    "User ID is a required field": "Användar-ID är ett obligatoriskt fält",
+    "Password is a required field": "Lösenord är ett obligatoriskt fält",
+    "Wrong username or password": "Ogiltigt användarnamn eller lösenord",
+    "Please sign in using 'Log in with Google'": "Logga in genom \"Google-inloggning\"",
+    "Password cannot be empty": "Lösenordet kan inte vara tomt",
+    "Password cannot be longer than 55 characters": "Lösenordet kan inte vara längre än 55 tecken",
+    "Please log in": "Logga inn",
+    "Invidious Private Feed for `x`": "Ogiltig privat flöde för `x`",
+    "channel:`x`": "kanal `x`",
+    "Deleted or invalid channel": "Raderad eller ogiltig kanal",
+    "This channel does not exist.": "Denna kanal finns inte.",
+    "Could not get channel info.": "Kunde inte hämta kanalinfo.",
+    "Could not fetch comments": "Kunde inte hämta kommentarer",
+    "View `x` replies": "Visa `x` svar",
+    "`x` ago": "`x` sedan",
+    "Load more": "Ladda flera",
+    "`x` points": "`x` poäng",
+    "Could not create mix.": "Kunde inte skapa mix.",
+    "Empty playlist": "Spellistan är tom",
+    "Not a playlist.": "Ogiltig spellista.",
+    "Playlist does not exist.": "Spellistan finns inte.",
+    "Could not pull trending pages.": "Kunde inte hämta trendsättande sidor.",
+    "Hidden field \"challenge\" is a required field": "Dolt fält \"challenge\" är ett obligatoriskt fält",
+    "Hidden field \"token\" is a required field": "Dolt fält \"token\" är ett obligatoriskt fält",
+    "Erroneous challenge": "Felaktig challenge",
+    "Erroneous token": "Felaktig token",
+    "No such user": "Ogiltig användare",
+    "Token is expired, please try again": "Token föråldrad, försök igen",
+    "English": "Engelska",
```

Comment:
In Swedish all the following names of languages are written in all lower case ;)

Reply:
I believe they're presented in a language picker, each its own option.

Do you mean: "We ought to turn them into lower-case variants" for that list?